sympy/assumptions/cnf.py

```diff
@@ -140,11 +140,18 @@ def __str__(self):
     __repr__ = __str__
 
 
-def to_NNF(expr, composite_map={}):
+def to_NNF(expr, decomposer=None):
```

Comment:
this is backwards incompatible -- does this matter?

Reply:
Can we just leave the args as they are but if modify documentation to indicate that if `composite_map` is `{}` then a full decomposition of the predicate will be supplied by `decompose_predicate`?

Reply:
This argument was introduced in 1.9 dev so it won't be backwards incompatible. But I'm open to any suggestion.